index.html

```diff
@@ -10,6 +10,11 @@
   <link rel='stylesheet' type='text/css' href='application.css'>
 </head>
 
+<div class='nav'>
```

Comment:
the <body> tag contains (or should contain) all visible information to the user, so you should put this inside the body, use a `header` tag instead of a `div`